src/tests/multihost/alltests/test_sssctl_local.py

```diff
@@ -85,7 +85,7 @@ def test_0002_bz1599207(self, multihost,
             assert 'Cache entry creation date' in \
                    cmd.stdout_text and cmd.returncode == 0
 
-    @pytest.mark.converted('test_sss_cache.py', 'test_sss_cache__cache_expire_message')
+    @pytest.mark.converted('test_cache.py', 'test_sss_cache__cache_expire_message')
```

Comment:
Is this still correct?  It looks like you moved test_sss_cache__cache_expire_message to test_tools__sss_cache_expired_does_not_print_unrelated_message.

Reply:
fixed.